### PR TITLE
Spec PR for `__traits(getCppNamespaces, sym)`

### PR DESCRIPTION
**Pull request by thewilsonator**
see https://github.com/dlang/dmd/pull/11863 for the corresponding dmd PR

### CODE REVIEW

File: spec/traits.dd [hunk 1]
```diff
@@ -1127,6 +1128,22 @@ void main()
...
 )
 
+$(H2 $(GNAME getCppNamespaces))
+	$(P The argument is a symbol.
+	The result is a tuple of strings, possibly empty, that correspond to the namespaces the symbol resides in.
+	)
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+extern(C++, "ns")
```
Comment (thewilsonator) [inline spec/traits.dd:1138]: SPEC_RUNNABLE_EXAMPLE_COMPILE insets a `void main(){}` around this code block resulting in `namespaces are not allowed here`, which macro should I use instead?

Comment (TurkeyMan) [reply spec/traits.dd:1138]: Test also with a non-string namespace:
```d
extern(C++, ns2) void fun();
```

Comment (TurkeyMan) [reply spec/traits.dd:1138]: Actually, if this is just for the spec, just make sure the unit tests for the implementation test with string and non-string namespace.

**Comment by Geod24** [comment]
@thewilsonator : Plz fix

**Comment by thewilsonator** [comment]
@Geod24 any ideas for https://github.com/dlang/dlang.org/pull/2867#discussion_r506046660 ? Otherwise I'll just axe it.

**Comment by Geod24** [comment]
Can't you do it without macro ? Using `---` will give you a code block.

**Comment by thewilsonator** [comment]
Should hopefully be green now.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**